### ISSUE TITLE
Provide new API method to get list of visitor IDs

### ISSUE THREAD

**Issue by sgiehl** [piwik/piwik#11863]
Provide a new API method `Live.getLastVisitorIDs` (or similar), that returns a list of the latest visitor IDs (maybe sorted by the date of their last visit as default)

The API method should fully support segmentation, so it is possible to search for specific users.

**Comment by mattab** [comment]
Note it is already possible to get the list of Visitor IDs with support for segments, by leveraging the existing API `?module=API&method=Live.getLastVisitsDetails` and appending `&showColumns=visitorId`

**Comment by sgiehl** [comment]
Correct. But that report contains duplicate visitor IDs. Also it processes the complete visit details which is not necessary and takes quite long.
